engine/standalone/util/src/main/scala/pl/touk/nussknacker/engine/standalone/utils/typed/TypedMapUtils.scala

```diff
@@ -0,0 +1,30 @@
+package pl.touk.nussknacker.engine.standalone.utils.typed
+
+import io.circe.{Json, JsonObject}
+import pl.touk.nussknacker.engine.api.typed.TypedMap
+
+import scala.collection.JavaConverters._
+
+object TypedMapUtils {
+
+    def jsonToTypedMap(json: Json): TypedMap =
```

Comment:
Where do we need/use this?

Reply:
I am working on tests to demonstrate how and when we need to use it. It is going to be a source which expect JSON in request body (schema defined in source field). This utils helps to change `Json` to `TypedMap`

Reply:
I've added `JsonSchemaStandaloneSourceFactory` which uses those utils

Reply:
maybe let's move it to util? so that we can replace 
`pl.touk.nussknacker.engine.kafka.generic.source.jsonToMapn` with it in the future?

Reply:
I have moved (and renamed) this file to utils